ts/components/ButtonDefaultOpacity.tsx

```diff
@@ -27,22 +33,42 @@ const getSlopForCurrentButton = (props: Props) => {
 /**
  * return Button component where the activeOpacity is 1.0 by default
  * instead of 0.2 https://github.com/facebook/react-native/blob/3042407f43b69994abc00350681f1f0a79683bfd/Libraries/Components/Touchable/TouchableOpacity.js#L149
+ *
+ * In certain cases on Android devices, the native-base button doesn't dispatch the onPress event
+ * Because of this, on Android, we wrap the button in a TapGestureHandler and the onPress is handled manually
+ * (this surely happen when a button is used inside a BottomSheet)
  */
 const ButtonDefaultOpacity = (props: Props) => {
   const slop = getSlopForCurrentButton(props);
-  return (
+  const button = (
     <Button
       {...{
         ...props,
         activeOpacity: props.activeOpacity || defaultActiveOpacity
       }}
+      onPress={isIos ? props.onPress : undefined}
```

Comment:
```suggestion
      onPress={props.onPress}
```
Without this change I cannot login using the pinpad on Android.